EPRN.Portal/Areas/Exporter/Controllers/PRNSController.cs

```diff
@@ -231,6 +233,33 @@ public async Task<IActionResult> DecemberWaste(DecemberWasteViewModel decemberWa
                                     new { decemberWaste.Id });
         }
 
+        [HttpGet]
+        [ActionName(Routes.Areas.Actions.PRNS.DraftConfirmation)]
+        public async Task<IActionResult> DraftConfirmation(int? id)
+        {
+            if (id == null)
+                return NotFound();
+
+            var viewModel = await _prnService.GetDraftConfirmationViewModel(id.Value);
+            return View(viewModel);
```

Comment:
What happens if you land on this page when the PRN has already been created as a draft? I thought you were going to see to this

Reply:
Done.